docs/utxo-hd/utxo-db/utxo-db-api.tex

```diff
@@ -97,7 +99,7 @@ \subsection{In the consensus layer}
 to balance the resource use in all interactions between honest nodes and
 potentially dishonest nodes and thus resist DoS attacks. This design principle
 led us to an Ouroboros design that involves efficient access to recent ledger
-states. \cite{fake-stake} describe the consequences of the failure to adopt
+states. \citet{fake-stake} describe the consequences of the failure to adopt
```

Comment:
Shouldn't it be "describes"? :thinking:

Reply:
In the text "Kanjalkar et al. [2019] describe the consequence ...", it depends on whether you consider "Kanjalkar et al." to refer to the paper or its authors. I usually consider it to refer to the authors, so then the verb should be plural, i.e., "Kanjalkar et al. describe ..."